tempest2-testing-docker/build.gradle

```diff
@@ -0,0 +1,22 @@
+apply plugin: 'java-library'
+apply plugin: 'kotlin'
+
+dependencies {
+  api project(":tempest2-testing")
+  implementation project(":tempest2-testing-internal")
+  implementation dep.kotlinStdLib
+  // The docker-java we use in tests depends on an old version of junixsocket that depends on
+  // log4j. We force it up a minor version in packages that use it.
```

Comment:
Handy comment. Looking forward to the docker-java improvements that we may someday receive.